deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/ParserUtility.java

```diff
@@ -30,7 +33,10 @@
 {
     private static final String DATEFORMAT = "yyyy-MM-dd'T'HH:mm:ss";
     private static final String OFFSETFORMAT = "yyyy-MM-dd'T'HH:mm:ssXXX";
-    private static final String TIMEZONE = "UTC";
+
+    final static DateTimeFormatter UTC_DATETIME_FORMATTER = DateTimeFormatter.ofPattern(DATEFORMAT);
```

Comment:
We will still be returning date time strings in the same format as before, just without the change in hours that happened due to using local time zones.